### ISSUE TITLE
Github Workflow broken - FileNotFoundError: [Errno 2] No such file or directory: 'repo-plugins'

### ISSUE THREAD

**Issue by linqcan** [xbmc/kodi-addon-submitter#24]
Getting issues when I am using the submitter in Github Actions (after 15 month hiatus...).
```
addon-submitter - INFO: Getting addon.xml file content
addon-submitter - INFO: Creating ZIP file...
addon-submitter - INFO: ZIP created successfully.
addon-submitter - INFO: Getting addon.xml file content
urllib3.connectionpool - DEBUG: Starting new HTTPS connection (1): api.github.com:443
urllib3.connectionpool - DEBUG: https://api.github.com:443 "GET /repos/***/repo-plugins?type=all HTTP/1.1" 200 None
addon-submitter - INFO: Creating addon branch...
Traceback (most recent call last):
  File "/usr/local/bin/submit-addon", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.7/site-packages/addon_submitter/__main__.py", line 82, in main
    local_branch_name=local_branch_name
  File "/usr/local/lib/python3.7/site-packages/addon_submitter/utils.py", line [14](https://github.com/kodi-svtplay/xbmc-svtplay/runs/5833075157?check_suite_focus=true#step:4:14)5, in create_addon_branch
    os.chdir(repo)
FileNotFoundError: [Errno 2] No such file or directory: 'repo-plugins'
```

https://github.com/kodi-svtplay/xbmc-svtplay/runs/5833075157?check_suite_focus=true

Could you help me out @enen92 ? 🙏

**Comment by romanvm** [comment]
As it turned during with my attempts to submit the new TVDB scrapers, the submitter script does not support submissions from organizations (not individual GH users). It definitely needs to be addressed.

**Comment by linqcan** [comment]
> As it turned out during with my attempts to submit the new TVDB scrapers, the submitter script does not support submissions from organizations (not individual GH users). It definitely needs to be addressed.

Not sure I follow @romanvm , I am submitting as a user (albeit functional in our organization) and it has worked previously from Github Actions (for me).

**Comment by linqcan** [comment]
@romanvm while looking at the logs I see that the workflow I am running is using your fork of this repo, why is that? 🤔 
![image](https://user-images.githubusercontent.com/832984/161816617-3de066f8-8e35-4a0d-a405-027fd9b098b0.png)

**Comment by linqcan** [comment]
After some debugging I've found that the reason for the script failing is:
![image](https://user-images.githubusercontent.com/832984/161828025-e53ecf44-3fd3-4883-b33e-fd4f91941cc1.png)

**Comment by linqcan** [comment]
I have a fix incoming, PR soon.

**Comment by linqcan** [comment]
Turns out @mediaminister already had solved this in pull #23 , can someone merge either of them to get us running again?

**Comment by linqcan** [comment]
Fixed by #23
